### PR TITLE
Feature/email form

### PR DESCRIPTION
**Pull request by AIbrahimQA**

### CODE REVIEW

**Review by JoNB7777 — APPROVED** [review]

File: app/models/EmailForm.scala [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+import play.api.data.Forms._
+import play.api.libs.json.OFormat
+
+
+case class EmailForm(email: String, subject: String, message: String)
+
+object EmailForm {
+  val email: Form[EmailForm] = Form(
+    mapping(
+      "email" -> nonEmptyText,
```
Comment (DannyPennington) [inline app/models/EmailForm.scala]:
```suggestion
      "email" -> email,
```

Comment (DannyPennington) [reply app/models/EmailForm.scala]: In the brief validating the email is one of the requirements

**Review by DannyPennington — CHANGES_REQUESTED** [review]
Looking good, just one minor change from me

File: app/models/EmailForm.scala (hunk 1 above)
Comment (AIbrahimQA) [reply app/models/EmailForm.scala]: Done

**Review by DannyPennington — APPROVED** [review]

**Pull request merged**